File: app/src/main/java/com/gp/mldview/languagelib/BaseActivity.java
package com.gp.mldview.languagelib;

import android.content.Context;
import android.content.res.Configuration;
import android.util.Log;


import androidx.appcompat.app.AppCompatActivity;

import java.util.Locale;

public class BaseActivity extends AppCompatActivity {

    protected String TAG = getClass().getSimpleName();

    @Override
    protected void attachBaseContext(Context newBase) {
        Log.e(TAG, "attachBaseContext");
        super.attachBaseContext(MultiLanguageUtil.attachBaseContext(newBase));
        Locale locale = getResources().getConfiguration().locale;
        Log.e(TAG, "attachBaseContext locale " + locale  + "  getLanguageLocale " + MultiLanguageUtil.getInstance().getLanguageLocale(this));

    }

    @Override
    public void onConfigurationChanged(Configuration newConfig) {
        Log.e(TAG, "onConfigurationChanged");
        super.onConfigurationChanged(newConfig);
    }
}

File: app-version1/src/main/java/com/chain/app_version1/BaseActivity.kt
package com.chain.app_version1

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import dagger.android.AndroidInjection

open class BaseActivity: AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        AndroidInjection.inject(this)
        super.onCreate(savedInstanceState)
    }
}